### PR TITLE
Internationalization: Translate TimeRangeList component

### LINKED ISSUE #52943: Translate TimeRangeList
As the #52648 issue is more complex than we initially thought, we divide it into smaller tasks. This task is focused on translating the TimeRangeList component.

Question: The texts of the TimeRangeList options are in _packages > grafana-data > src > datetime > rangeutil.ts_. How can we translate them? Can we do it now or is it a problem to be solved in the near future?

### PR DESCRIPTION
**Pull request by eledobleefe**
**What this PR does / why we need it**:
As the https://github.com/grafana/grafana/issues/52648 issue is more complex than we initially thought, we divide it into smaller tasks. This task is focused on translating the TimeRangeList component (it involves also the TimePickerContent, the TimeRangePicker and the DashNavTimeControls components).

**Which issue(s) this PR fixes**:

Fixes #52943

**Special notes for your reviewer**: The texts of the TimeRangeList options are in _packages > grafana-data > src > datetime > rangeutil.ts_. How can we translate them? Can we do it now or is it a problem to be solved in the near future?

### CODE REVIEW

File: packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx [hunk 1]
```diff
@@ -43,6 +43,15 @@ export interface TimeRangePickerProps extends Themeable {
   history?: TimeRange[];
   hideQuickRanges?: boolean;
   widthOverride?: number;
+  //Internationalization
+  timeRangeListTitleMessage?: string;
+  timeRangeListRoleDescriptionMessage?: string;
+  timeRangeListEmptyFirstMessage?: string;
+  timeRangeListEmptyLinkMessage?: string;
```
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx:50]: This one is no longer needed

File: packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx [hunk 2]
```diff
@@ -43,6 +43,15 @@ export interface TimeRangePickerProps extends Themeable {
...
+  timeRangeListEmptySecondMessage?: (
+    stylesEmptyList: string | undefined,
```
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx:52]: The name of this one has changed now

**Review by JoaoSilvaGrafana — COMMENTED** [review]
Great job with this very complicated one 👍 Two small comments.
Might leave this one open until our brainstorm Monday as well and we can decide if we want to merge it or change it post brainstorm. 🙏

**Comment by eledobleefe** [comment]
> Great job with this very complicated one 👍 Two small comments. Might leave this one open until our brainstorm Monday as well and we can decide if we want to merge it or change it post brainstorm. 🙏

@JoaoSilvaGrafana, maybe it is good to stop working on TimePickerRange internationalization until the brainstorming, isn't it? I'll make the changes you've suggested and wait until the meeting 👌

**Pull request closed without merging**